crates/forge-runner/src/test_case_summary.rs

```diff
@@ -3,13 +3,49 @@ use crate::expected_result::{ExpectedPanicValue, ExpectedTestResult};
 use cairo_felt::Felt252;
 use cairo_lang_runner::short_string::as_cairo_short_string;
 use cairo_lang_runner::{RunResult, RunResultValue};
+use num_integer::Roots;
 use std::option::Option;
 
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Clone, Default)]
 pub struct GasStatistics {
     pub min: u128,
     pub max: u128,
+    pub maybe_mean: Option<u128>,
+    pub maybe_std_deviation: Option<u128>,
```

Comment:
```suggestion
    pub mean: Option<u128>,
    pub std_deviation: Option<u128>,
```
Type already contains this information